lib/opal/electron/browser_window.rb

```diff
@@ -9,8 +9,26 @@ def initialize(name = 'main_window', params = {}, debug: false)
       @native = JS.new(`BrowserWindow`, `params.$to_n()`)
       @native.JS.loadURL("file://#{`__dirname`}/#{name}.html")
       @native.JS[:webContents].JS.openDevTools if debug
+      methods_ruby = []
+      %x{
+        for(var method in #@native) {
+          #{methods_ruby << `method`}
```

Comment:
note that we create a new array of strings in ruby to avoid bugs that occur when we call method[0] directly